### PR TITLE
Synth_comp

### PR DESCRIPTION
**Pull request by profxj**
Some small edits for synthesizing components with lines that
don't contribute to the column.

And a test modification

### CODE REVIEW

File: linetools/isgm/abscomponent.py [hunk 1]
```diff
@@ -396,6 +396,10 @@ def synthesize_colm(self, overwrite=False, redo_aodm=False, **kwargs):
         # Collate
         self.flag_N = 0
         for aline in self._abslines:
+            if aline.attrib['flag_N'] == 0:  # No value
+                warnings.warn("Absline {} has flag=0.  Hopefully you expected that")
```
Comment (nhmc) [inline linetools/isgm/abscomponent.py]: Did you mean to print the absorption line info here? There is a '{}' but no argument to print.

**Comment by nhmc** [comment]
Apart from the inline comment, this looks great. Please merge when ready.

File: linetools/isgm/abscomponent.py (hunk 1 above)
Comment (profxj) [inline linetools/isgm/abscomponent.py]: Indeed!

**Comment by profxj** [comment]
Merging.

**Pull request merged**